app/views/homes/show.html.erb

```diff
@@ -72,5 +72,12 @@
     </table>
   </p>
 
+  <!--
+    /// DO NOT REMOVE ///
+    /// This counter is from http://www.webestools.com/visitors-counter-free-number-unique-visitors-users-visits-statistics.html ///
+    /// DO NOT REMOVE ///
+  -->
+  <div style="text-align:center;"><script type="text/javascript" src="http://services.webestools.com/cpt_visitors/40122-13-5.js"></script></div><a href="http://www.webestools.com/" style="display:block;text-align:center;" title="Tools services webmasters counters generators scripts tutorials free"></a>
```

Comment:
I love this feature.

On Friday, July 15, 2016, Edward Loveall <notifications@github.com> wrote:

> In app/views/homes/show.html.erb
> <https://github.com/gabebw/hotline-webring/pull/52#discussion_r70973197>:
>
> > @@ -72,5 +72,12 @@
> >      </table>
> >    </p>
> >
> > +  <!--
> > +    /// DO NOT REMOVE ///
> > +    /// This counter is from http://www.webestools.com/visitors-counter-free-number-unique-visitors-users-visits-statistics.html ///
> > +    /// DO NOT REMOVE ///
> > +  -->
> > +  <div style="text-align:center;"><script type="text/javascript" src="http://services.webestools.com/cpt_visitors/40122-13-5.js"></script></div><a href="http://www.webestools.com/" style="display:block;text-align:center;" title="Tools services webmasters counters generators scripts tutorials free"></a>
>
> Maybe instead of the comment, put a class on the div like visitor-counter
> to give future devs an idea of what it is? It even has documentation built
> in linking back to the site.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/gabebw/hotline-webring/pull/52/files/e1d16c2d310985a2b857e85c5d2d325754cdb3f0#r70973197>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AA3xXVPNFuSVvDEoIUBcqwSGQeh_1NOOks5qV4xbgaJpZM4JNIyC>
> .
>


-- 


-- 
@adarshp <http://www.twitter.com/adarshp>
313.454.1515 (mobile)
See when I am free: http://bit.ly/adarshcal